build/PlatformInstaller.targets

```diff
@@ -0,0 +1,300 @@
+      OverwriteReadOnlyFiles="True"
+      SkipUnchangedFiles="False"
+      UseHardlinksIfPossible="False" />
+
+    <ItemGroup>
+      <DebConfigItems Include="MAINTAINER_NAME" Replacement="$(MaintainerName)" />
+      <DebConfigItems Include="MAINTAINER_EMAIL" Replacement="$(MaintainerEmail)" />
+      <DebConfigItems Include="HOMEPAGE" Replacement="$(Homepage)" />
+      <DebConfigItems Include="INSTALL_ROOT" Replacement="$(InstallRoot)" />
+      <DebConfigItems Include="PACKAGE_NAME" Replacement="$(DebPrefix)" />
+      <DebConfigItems Include="PACKAGE_REVISION" Replacement="$(PackageRevision)" />
+      <DebConfigItems Include="LICENSE_TYPE" Replacement="$(LicenseType)" />
+      <DebConfigItems Include="SHORT_DESCRIPTION" Replacement="$(DebSummary)" />
+      <DebConfigItems Include="LONG_DESCRIPTION" Replacement="$(DebDescription)" />
+      <DebConfigItems Include="DEBIAN_DEPENDENCIES" Replacement="$(DebDependencies)" />
+    </ItemGroup>
+
+    <!-- Update versions -->
+    <RepoTasks.ReplaceInFile Filename="$(_WorkLayoutDir)debian_config.json" Items="@(DebConfigItems)" />
+
+    <!-- Build SF and Hosting Bundle Deb package -->
+
+    <!--
+      Note: KOREBUILD_DOTNET_FEED_CREDENTIAL is intentionally NOT an MSBuild variable.
+      MSBuild doesn't to the substitution correctly because the string contains %,
+      so we'll let bash do it instead.
+    -->
+
+    <Exec Command="docker run
+      --rm
+      -v $(RepositoryRoot):$(_DockerRootDir)
+      -e DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true
+      -e INSTALLER_NAME=$(DebPrefix)-$(DebVersion)
+      -e INSTALLER_VERSION=$(DebVersion)
+      -e 'KOREBUILD_DOTNET_VERSION=$(KOREBUILD_DOTNET_VERSION)'
+      -e 'KOREBUILD_DOTNET_SHARED_RUNTIME_VERSION=$(KOREBUILD_DOTNET_SHARED_RUNTIME_VERSION)'
+      -e 'KOREBUILD_DOTNET_FEED_CDN=$(KOREBUILD_DOTNET_FEED_CDN)'
+      -e 'KOREBUILD_DOTNET_FEED_UNCACHED=$(KOREBUILD_DOTNET_FEED_UNCACHED)'
+      -e &quot;KOREBUILD_DOTNET_FEED_CREDENTIAL=$KOREBUILD_DOTNET_FEED_CREDENTIAL&quot;
+      docker-image-$(Image)
+      ./build.sh /t:RunDebTool"
+      ContinueOnError="WarnAndContinue" />
+
+    <!-- Copy SF and Hosting Bundle packages to output -->
+    <ItemGroup>
+      <GeneratedDebFiles Include="$(_WorkOutputDir)/*.deb" />
+    </ItemGroup>
+
+    <Error Text="@(GeneratedDebFiles->Count()) deb installer files generated." Condition="'@(GeneratedDebFiles->Count())' != 1" />
+
+    <Copy
+      DestinationFiles="$(_InstallersOutputDir)$(DebPrefix)-$(DebVersion)-$(Image)-x64.deb"
+      SourceFiles="@(GeneratedDebFiles)"
+      OverwriteReadOnlyFiles="True"
+      SkipUnchangedFiles="False"
+      UseHardlinksIfPossible="False" />
+  </Target>
+
+  <Target Name="_GenerateDebOnPlatform">
+    <!-- Build Docker Image -->
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_BuildDockerImage" Properties="Image=$(Image)" />
+
+    <ItemGroup>
+      <_DebSFDependencies Include="aspnetcore-store-2.0.3"/>
```

Comment:
Likewise, add a comment here why 2.1 depends on 2.0.x stuff.

Also, we should centralize the version as a variable in dependencies.props. Something like `MicrosoftAspNetCore20RuntimeStoreVersion`?

Reply:
Just going to hardcode the versions for now. This is a temporary dependency anyway.